src/libs/API/parameters/AddCommentOrAttachementParams.ts

```diff
@@ -1,7 +1,7 @@
 import type {FileObject} from '@components/AttachmentModal';
 
 type AddCommentOrAttachementParams = {
-    reportID: string;
+    reportID?: string;
```

Comment:
I don't think it makes sense to allow this to be undefined because we will throw an error on the backend if the report can't be found for AddComment. Instead let's return early and log a warning if reportID is undefined when setting up these params. It should never happen and if it does we'll be able to investigate why.